gbpservice/neutron/tests/unit/nfp/proxy_agent/modules/test_proxy_agent.py

```diff
@@ -35,49 +35,62 @@ def setUp(self):
         self.manager = rpc_manager('conf', 'sc')
         self.context = TestContext().get_context()
         self.import_lib = 'gbpservice.nfp.lib.transport'
+        self.body = {"info": {"context": {}},
+                     "config": []}
 
     def _post(self, conf, context, body,
               method_type, device_config=False,
               network_function_event=False):
         return (200, '')
 
     def test_create_network_function_config(self):
-        _data = "data"
```

Comment:
Data format access is changed in proxy_agent.py. It is expecting body in request data format. This change was introduced because of log meta data.